cluster/gce/gci/configure-helper.sh

```diff
@@ -1269,9 +1269,14 @@ EOF
 function create-kubeconfig {
   local component=$1
   local token=$2
-  echo "Creating kubeconfig file for component ${component}"
+  local path="/etc/srv/kubernetes/${component}/kubeconfig"
   mkdir -p "/etc/srv/kubernetes/${component}"
-  cat <<EOF >"/etc/srv/kubernetes/${component}/kubeconfig"
+
+  if [[ -e "${KUBE_HOME}/bin/gke-internal-configure-helper.sh" ]]; then
+    gke-internal-create-kubeconfig "${component}" "${token}" "${path}"
```

Comment:
Ditto the comment from `#100764` -  does the existence of `*configure-helper.sh` mean we've sourced it here?  I'm concerned about `command-not-found` errors, though there seems to be a lot of assumption throughout this script about the state of the `PATH`.

Reply:
It's sourced in https://github.com/Kubernetes/kubernetes/blob/master/cluster/gce/gci/configure.sh#L386